src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -111,6 +112,18 @@ void set(final Event e, final String message) {
 					((PlayerQuitEvent) e).setQuitMessage(message);
 			}
 		},
+		UNKNOWN("unknown command", "unknown command( |-)message", UnknownCommandEvent.class) {
+			@Override
+			@Nullable
+			String get(final Event e) {
+				return ((UnknownCommandEvent) e).getMessage();
+			}
+
+			@Override
+			void set(final Event e, final String message) {
+				((UnknownCommandEvent) e).setMessage(message);
```

Comment:
Since you're at it, remove the final modifiers from this code only and use `event` instead of `e`